code/game/objects/items/devices/traitordevices.dm

```diff
@@ -248,3 +248,103 @@
 	charges = 8
 	max_charges = 8
 	flawless = TRUE
+
+/obj/item/batterer
+	name = "mind batterer"
+	desc = "A dangerous syndicate device focused on crowd control and escapes. Causes brain damage, confusion, and other nasty effects to those surrounding the user."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "batterer"
+	throwforce = 5
+	w_class = WEIGHT_CLASS_TINY
+	throw_speed = 4
+	throw_range = 10
+	flags = CONDUCT
+	item_state = "electronic"
+	origin_tech = "magnets=3;combat=3;syndicate=3"
+
+	/// How many uses the mind batter has been used
+	var/times_used = 0
+	var/max_uses = 5
+	/// Is this item on cooldown from being thrown
+	var/on_throwing_cooldown = FALSE
+
+/obj/item/batterer/examine(mob/user)
+	. = ..()
+	. += "<span class='warning'>A little label on the side reads: \"Warning: Using this item in quick succession may cause fatigue to the user!\"</span>"
+	if(times_used >= max_uses)
+		. += "<span class='notice'>[src] is out of charge.</span>"
+	if(times_used < max_uses)
+		. += "<span class='notice'>[src] has [max_uses-times_used] charges left.</span>"
+
+/obj/item/batterer/attack_self(mob/living/carbon/user)
+	activate_batterer(user)
+
+/obj/item/batterer/proc/activate_batterer(mob/user)
+	if(user)
+		if(times_used >= max_uses)
+			to_chat(user, "<span class='danger'>The mind batterer has been burnt out!</span>")
+			return
+		if(!do_after_once(user, 2 SECONDS, target = src, allow_moving = TRUE, attempt_cancel_message = "You think it's best to save this for later."))
+			return
+
+	for(var/mob/living/M in oview(7, get_turf(src)))
+		if(issilicon(M))
+			M.Weaken(10 SECONDS)
+		else
+			M.Confused(30 SECONDS)
+		M.adjustBrainLoss(10)
+		to_chat(M, "<span class='danger'>You feel a sudden, electric jolt travel through yourself,</span>")
```

Comment:
```suggestion
		to_chat(M, "<span class='danger'>You feel a sudden, electric jolt travel through yourself, </span>")
```
you need a space if you're doing multi-part messages like this, right?

Reply:
No